src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    abstract String printFigureType();

```

Comment:
The same. Let's rename it.

Moreover, think: do you actually need this method? You are using it only within the `draw()`. Can you simplify it?